pkg/operation/botanist/component/nginxingress/nginxingress.go

```diff
@@ -412,6 +412,9 @@ func (n *nginxIngress) computeResourcesData() (map[string][]byte, error) {
 					v1beta1constants.LabelApp: labelAppValue,
 					labelKeyComponent:         labelValueController,
 				},
+				Annotations: map[string]string{
+					references.AnnotationKey(references.KindConfigMap, configMap.Name): configMap.Name,
```

Comment:
Can you add a comment here explaining that the `InjectAnnotations` function is not used since the `ConfigMap` is not mounted as volume (i.e., using the function wouldn't have any effect)?